### PR TITLE
Get-EntraApplicationTemplate

### PR DESCRIPTION
**Pull request by v-uansari**
All example are working.

### CODE REVIEW

File: module/Entra/AdditionalFunctions/Get-EntraApplicationTemplate.ps1 [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+function Get-EntraApplicationTemplate {
+    [CmdletBinding(DefaultParameterSetName = 'GetQuery')]
+    param (
+    [Parameter(ParameterSetName = "GetById", Mandatory = $true, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.String] $Id
```
Comment (KenitoInc) [inline module/Entra/AdditionalFunctions/Get-EntraApplicationTemplate.ps1:8]: We could support more parameters

-Property
We can select specific properties
image

-Top
image

Comment (v-pughosh) [reply module/Entra/AdditionalFunctions/Get-EntraApplicationTemplate.ps1]: added the paramters

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationTemplate.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+schema: 2.0.0
+---
+
+# Get-EntraApplicationTemplate
+
+## Synopsis
+
+Retrieve a list of applicationTemplate objects.
+
+## Syntax
```
Comment (KenitoInc) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationTemplate.md:25]: Add `Top` and `Property` params

File: test/module/Entra/Get-EntraApplicationTemplate.Tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        "homePageUrl" = "samplehomePageUrl"
+        "logoUrl" = "samplelogourl"
+        "categories" = @{}
+        "description" = ""
+        "supportedProvisioningTypes" = @{}
+    }
+
+    Mock -CommandName Invoke-GraphRequest -MockWith { $response } -ModuleName Microsoft.Graph.Entra
+}
+Describe "Get-EntraApplicationTemplate tests"{
```
Comment (KenitoInc) [inline test/module/Entra/Get-EntraApplicationTemplate.Tests.ps1:24]: Add tests for `Top` and `Property` params

Comment (snehalkotwal) [reply test/module/Entra/Get-EntraApplicationTemplate.Tests.ps1:24]: Done changes.

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationTemplate.md (hunk 2 above)
Comment (snehalkotwal) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationTemplate.md:25]: Done changes.

File: test/module/Entra/Get-EntraApplicationTemplate.Tests.ps1 (hunk 3 above)
Comment (v-pughosh) [reply test/module/Entra/Get-EntraApplicationTemplate.Tests.ps1:24]: added

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**